newsletter-december-2022.rst

```diff
@@ -0,0 +1,132 @@
+.. post:: Dec 5, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - December 2022
+========================================
+
+Here are the first features and updates that have hatched since we announced a Q4 focus on core platform features in the :doc:`previous newsletter </newsletter-december-2022>`.
+
+
+News and updates
+----------------
+
+The latest updates from our team:
+
+- 🐘️ We started *tooting* in the Fediverse (Mastodon). Follow us at `@readthedocs@fosstodon.org <https://fosstodon.org/@readthedocs>`_
+- 🎬️ Visit the upcoming landing page `about.readthedocs.com <https://about.readthedocs.com/>`_ and `let us know <mailto:hello@readthedocs.org>`_ know if you have comments or ideas for it.
+- ⚙️ The *entire* build process can now be overridden (if you need it). :doc:`See the Announcement </build-customization>`.
```

Comment:
This should be up top.

Reply:
Oh that's a very very good suggestion, will do that after adding changes :+1:

Reply:
Moved to the top :+1: